src/elements/content-sidebar/SidebarClassification.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @flow
+ * @file Classification sidebar component
+ * @author Box
+ */
+
+import React from 'react';
+import getProp from 'lodash/get';
+import { FormattedMessage, injectIntl } from 'react-intl';
+
+import Classification from '../../features/classification';
+import messages from '../common/messages';
+import { INTERACTION_TARGET, SECTION_TARGETS, DETAILS_TARGETS } from '../common/interactionTargets';
+import Collapsible from '../../components/collapsible';
+import PlainButton from '../../components/plain-button/PlainButton';
+import IconEdit from '../../icons/general/IconEdit';
+import { FIELD_PERMISSIONS_CAN_UPLOAD } from '../../constants';
+import './SidebarClassification.scss';
+
+type Props = {
+    classification: ClassificationInfo,
+    file: BoxItem,
+    onEdit?: () => void,
+} & InjectIntlProvidedProps;
+
+const SidebarClassification = ({ classification, file, intl, onEdit }: Props) => {
+    const isEditable = !!onEdit && getProp(file, FIELD_PERMISSIONS_CAN_UPLOAD, false);
+    const hasClassification = !!getProp(classification, 'name');
```

Comment:
Is `getProp` needed here? Can `classification` be null/undefined?

Reply:
Yeah, since it may not be passed in at all or the feature doesn't exist or feature exists but no classification was added

Reply:
I fixed the flowtype to have ?